src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/DisableRuntimeMarshallingTests.cs

```diff
@@ -277,6 +487,45 @@ struct ValueType2
             await VerifyCSAnalyzerAsync(source);
         }
 
+        [Fact]
+        public async Task VB_PInvokeWithUnmanagedValueTypeParameter_WithAutoLayoutField_Emits_Diagnostic()
+        {
+            string source = @"
+Imports System.Runtime.CompilerServices
+Imports System.Runtime.InteropServices
+<assembly:DisableRuntimeMarshalling>
+Public Class Foo
```

Comment:
I just realized that policheck will flag `Foo`. Can we rename all the `Foo` and `foo` strings in this file to something else?